app/src/main/java/com/fmpdroid/moneybox/ScreenSlidePageFragment.java

```diff
@@ -38,6 +63,53 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
             tvTargetDate.setText(moneybox.getTargetDate());
         }
 
+        editTextAmount.addTextChangedListener(new TextWatcher() {
+            @Override
+            public void beforeTextChanged(CharSequence charSequence, int i, int i1, int i2) {
+            }
+
+            @Override
+            public void onTextChanged(CharSequence charSequence, int i, int i1, int i2) {
+                String textAmount = charSequence.toString().trim().length() > 0 ? "Amount: " + charSequence.toString() : "Amount: 0";
```

Comment:
```suggestion
                String textAmount = "Amount: ";
                if (charSequence.toString().trim().length() <= 0) { 
                     textAmount  += "0";
                }
```

Do not always jump to ternary checking.